content/basics/ably.textile

```diff
@@ -0,0 +1,90 @@
+---
+title: What is Ably?
+meta_description: "A brief introduction to Ably and its features."
+---
+
+Ably’s realtime platform provides  a global realtime pub/sub network, that is scalable and fault-tolerant through our Realtime or REST interfaces. Built on the "Four Pillars of Dependability":#infrastructure Ably provides reliable performance available when needed.
+
+h2(#what). What can you build with Ably?
+
+Ably is used across a variety of industries to provide scalable building realtime collaborative web experiences, such as:
```

Comment:
Add 5 core use cases here instead of below: 
* Chat applications
* Syncing data across devices
* Broadcasting realtime even data across devices
* Push notifications
* Collaborative multiplayer experiences